Automation-Networks.md

```diff
@@ -2,11 +2,53 @@
 
 This section will describe **Automation Networks** in Cacti.
 
-![Automation Networks](images/automation-networks.png)
+Adding a network to scan in the automation plugin is easy  on the main console click Automation
+once on the below page click the + on the top right of the page 
+
+![Automation Networks](images/automation-network-main.png)
+
+<br>
+<br>
+<br>
+
+you will now see the below page for example if you want to scan 192.168.1.0/24 you would enter that in the subnet range textbox<br>
+you will need to note the subnet in CIDR format<br>
+
+Other important options are 
+<br>
+Schedule type - How often you want to scan this subnet for devices<br>
```

Comment:
1. Add a - (minus) to the beginning of the line to denote a bullet list
   or convert to an MDL table
2. Do not use inline HTML codes

Reply:
Sounds Good! 

I am ok with those changes